Analysis_of_rich_people.R

```diff
@@ -22,5 +76,33 @@ plot <- ggplot(top_10_billionaires, aes(x = reorder(personName, -finalWorth), y
        fill = "Billionaire") +
   theme(axis.text.x = element_text(angle = 45, hjust = 1))
```

Comment:
VIelleicht bei dem Chart noch die tatsächlichen Werte in die Bars reinschreiben, wenn wir das Diagramm behalten wollen